enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -79,7 +79,12 @@ def __init__(self, kernel_manager, proxy_config):
             self.log.debug("{class_name} shutdown wait time adjusted to {wait_time} seconds.".
                            format(class_name=type(self).__name__, wait_time=kernel_manager.shutdown_wait_time))
 
+        self.yarn_resource_check_wait_time = 0.20 * self.kernel_launch_timeout
```

Comment:
Let's add a comment stating we're going to take up 20% of the kernel launch timeout to check on resource availability.

Reply:
Added